docs/units/index.rst

```diff
@@ -254,11 +254,6 @@ Reference/API
 
 .. automodapi:: astropy.units.equivalencies
 
-.. automodapi:: astropy.units.function
```

Comment:
why are these removed? They are distinct from `astropy.units.function.units` and, e.g., `STmag` does not seem to be linked anywhere else on the present doc pages.

Reply:
I think I had to remove this one as well to fix the warnings.

Reply:
And how about `function.logarithmic`? Is removing that truly needed? If present, what is the error?